cdc/sink/codec/avro/avro.go

```diff
@@ -194,7 +194,7 @@ func (a *BatchEncoder) avroEncode(
 	avroCodec, registryID, err := schemaManager.GetCachedOrRegister(
 		ctx,
 		topic,
-		e.TableInfoVersion,
+		e.TableInfo.TableInfoVersion,
```

Comment:
how about rename the `TableInfoVersion` which is quite mouthful to `Version` directly

Reply:
Done